### DISCUSSION TITLE
Trying to create a collision event on a basic box

### DISCUSSION THREAD

**Discussion by luddensdesir** [enable3d/enable3d#269] [Help / Questions]
This is how I make my box:

```
    const box = this.physics.add.box(
      { x: 0, y: 2, z: -5, width: 1, height: 1, depth: 3, mass :1, collisionFlags: 0 },
      { lambert: { color: 'red', transparent: true, opacity: 0.5 } }
    );
```

But I can't get collision events working properly on it. What I'm trying to do is on collide dramatically lower the amount of force that can be applied so that my object stops humping everything it bumps into and just bounces off.

**Comment by yandeu** [comment]
Have you looked at the example [Collision Detection](https://enable3d.io/examples/collision-detection.html)?

**Comment by bleuscyther** [comment]
When dealing with that I play with the `.body.setBounciness` or `.body.setRestitution` of the objects colliding
